index.js

```diff
@@ -57,11 +82,14 @@ exports.parse = function (str) {
 exports.stringify = function (obj, opts) {
 	var defaults = {
 		encode: true,
-		strict: true
+		strict: true,
+		arrayFormat: 'default'
```

Comment:
`none` sounds good. `repeat` well... For sure it has some science behind their choice but i just can find it... But, it's up to you :-) i'm fine with any. The default value is the fallback so probably nobody's gonna write it.